src/Uno.UWP/ApplicationModel/DataTransfer/StandardDataFormats.cs

```diff
@@ -1,15 +1,19 @@
+#nullable enable
+
 #if !NET461
 namespace Windows.ApplicationModel.DataTransfer
 {
-	public partial class StandardDataFormats 
+	public partial class StandardDataFormats
 	{
-		public static string Text => "Text";
-
-#if __ANDROID__
-		public static string Html => "HTML Format";
-		public static string Uri => "UniformResourceLocatorW";
-		public static string WebLink => "UniformResourceLocatorW";
-#endif
+		public static string UserActivityJsonArray { get; } = "UserActivityJsonArray";
+		public static string WebLink               { get; } = "UniformResourceLocatorW";
+		public static string ApplicationLink       { get; } = "ApplicationLink";
+		public static string Text                  { get; } = "Text";
+		public static string Uri                   { get; } = "UniformResourceLocatorW";
```

Comment:
Codacy found an issue: [Change this property type to 'System.Uri'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6333952)